src/store/types.ts

```
@@ -0,0 +1,52 @@
+  RecentState,
+  VideoProgress,
+  VideoProgressState,
+} from '../types';
+
+// Store model interfaces
+export interface PlaylistModel extends PlaylistState {
+  setLoading: Action<PlaylistModel, boolean>;
+  setError: Action<PlaylistModel, string>;
+  addPlaylist: Action<PlaylistModel, Playlist>;
+  getPlaylist: Thunk<PlaylistModel, string>;
+  getPlaylistById: Thunk<PlaylistModel, string, {}, {}, Playlist>;
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)